test/daemon.spec.js

```diff
@@ -143,9 +144,16 @@ describe('daemon', () => {
 
     const response = Response.decode(await stream.first())
     expect(response.type).to.eql(Response.Type.OK)
+
+    const expectedMaddrs = daemon.libp2p.peerInfo.multiaddrs.toArray().map((ma) => {
+      const maStr = ma.toString().split('/ipfs/')[0]
+
+      return multiaddr(maStr).buffer
```

Comment:
You should be able to just do 
```js
return ma.decapsulate('ipfs')
```
It automatically removes everything after and including the last instance of `/ipfs/`

Reply:
Great, thanks @jacobheun !